### PR TITLE
Apps Framework updates

### PR DESCRIPTION
**Pull request by cwarnermm**
In preparation for the v7.0 launch in June, Docs is proposing updates to the Apps Framework developer documentation to reduce the barriers to entry for the greatest number of developers and integrators.

### CODE REVIEW

File: site/content/integrate/apps/_index.md [hunk 1]
```diff
@@ -1,41 +1,85 @@
...
+Mattermost Apps are lightweight, interactive add-ons to Mattermost implemented as a collection of HTTP endpoints. You can write Mattermost apps in your language of choice, and your app is supported on all Mattermost clients, including web, desktop, and mobile.
 
-![image](app-slash-command-zoomed-in.png)
+Apps are available as a Developer Preview and we're looking for your feedback! Share constructive feedback in the [Mattermost Apps channel](https://community.mattermost.com/core/channels/mattermost-apps) on our Mattermost Community instance.
 
-- Post messages to channels
+## What's the Mattermost Apps Framework?
 
-![image](app-bot.png)
+*[what is it and how does it differ from the existing plugin framework?]*
```
Comment (wiersgallak) [inline site/content/integrate/apps/_index.md:23]: App framework wraps the Mattermost API into a tool making it easy to develop integrations that interact with the Mattermost Server and Web App in predefined ways. 

They are different than plugins in that plugins have the ability to modify the Mattermost Server and Web App without having to fork those specific repositories.

File: site/content/integrate/apps/_index.md [hunk 2]
```diff
@@ -1,41 +1,85 @@
...
+    ```
+
+5. Deploy the Mattermost server by running the following command:
+
+    ```
+    sudo docker-compose -f docker-compose.yml -f docker-compose.without-nginx.yml up -d
+    ```
+6. Access your Mattermost workspace for the first time based on the hostname and port specified in the ENV file. Follow the onboarding flow as the first first System Admin user for this Mattermost workspace.
+
+## Try the test app
```
Comment (wiersgallak) [inline site/content/integrate/apps/_index.md:78]: Check out these sample apps: 
 - [ServiceNow](https://mattermost.com/marketplace/servicenow-app/) 
 - [Zendesk](https://mattermost.com/marketplace/zendesk-app/)

**Review by wiersgallak — CHANGES_REQUESTED** [review]
Added some comments/content for you to consider

**Comment by cwarnermm** [comment]
Closing this PR in favour of other PRs being merged into the code base.

**Pull request closed without merging**